src/cli/android/mod.rs

```diff
@@ -139,6 +140,15 @@ If you are interesting in library testing then please use advance mode with --li
         .into());
     }
 
+    if let Some(bundles) = &application_bundle {
```

Comment:
If a user added only `--test-application` without `application` then it is also unclear how the feature should work.

Reply:
Fixed